docs/dev_guide/ceph_s3_tests/ceph_s3_tests_pending_list_status.md

```diff
@@ -0,0 +1,64 @@
+# Ceph S3 Tests - Pending List Status
+
+## Introduction
+Ceph S3 tests are unofficial AWS S3 compatibility tests written in Python (for more information see: [ceph/s3-test](https://github.com/ceph/s3-tests) repository on GitHub) that we use in our CI.
+
+## Pending List Status
+Attached a table with tests that where investigated and their status (this table is partial).
+
+| Test (Partial Name)                                       | General Reason | Issue Number         | Additional Comments  |
+|-----------------------------------------------------------|-----------------|----------------------|----------------------|
+| test_bucket_create_exists                                 | Faulty Test     | [465](https://github.com/ceph/s3-tests/issues/465)                  |                      |
+| test_bucket_listv2_encoding_basic                         | Faulty Test     | [478](https://github.com/ceph/s3-tests/issues/478)                  |                      |
+| test_bucket_list_encoding_basic                           | Faulty Test     | [478](https://github.com/ceph/s3-tests/issues/478)                  |                      |
+| test_object_read_unreadable                               | Faulty Test     | [480](https://github.com/ceph/s3-tests/issues/480)                  |                      |
+| test_account_usage                                        | Not Supported   |                      | Noobaa list_buckets() don't have support for usage                     |
+| test_head_bucket_usage                                        | Not Supported   |                      | Noobaa list_buckets() don't have support for usage                     |
+| test_head_bucket_usage                                        | Not Supported   |                      | S3 test expecting Prefix inside rules not inside Filter, But in our code Prefix expected inside Filter                     |
```

Comment:
@naveenpaul1, Are those tests not passing because we need to change something in our code or there is a problem with the test itself?
In case those are faulty tests, I think it is better to open an issue with all the details (and add a link).

Reply:
@shirady ceph test `test_bucket_create_exists()` was failing because of the faulty test condition and @utkarsh-pro already created a PR for the same in ceph S3 test repo but has not merged yet.  https://github.com/ceph/s3-tests/pull/479, This is only one test I could find with ceph test issue.

Reply:
@naveenpaul1, this link was added, I thought that in the other cases where we have a comment that related to a faulty test to open an issue. As I understand from you the other comments are things that we need to fix in noobaa's code, right?